nucleus/grizzly/config/src/main/java/org/glassfish/grizzly/config/SSLConfigurator.java

```diff
@@ -187,7 +187,7 @@ protected SSLContext configureSSL() {
                 } else {
                     final String[] protocols = new String[tmpSSLArtifactsList.size()];
                     tmpSSLArtifactsList.toArray(protocols);
-                    enabledProtocols = protocols;
+                    this.setEnabledProtocols(protocols);
```

Comment:
Not sure the `this` is required?
```suggestion
                    setEnabledProtocols(protocols);
```